content/applications/inventory_and_mrp/inventory/product_management/product_tracking/package.rst

```diff
@@ -48,147 +44,85 @@ the :guilabel:`Operations` heading, activate the :guilabel:`Packages` feature. T
-   Warehouses`), the :guilabel:`Outgoing Shipments` option **must** be set to :guilabel:`Pack goods,
-   send goods in output and then deliver (3 steps)`.
-
-After confirming a sales order, selecting the :guilabel:`Delivery` smart button displays the three
-tasks to complete the delivery.
-
-.. image:: package/three-step.png
+.. figure:: package/destination-package.png
    :align: center
-   :alt: Clicking "Delivery" smart button displays the pick, pack, ship steps.
-
-To create a batch transfer, navigate to :menuselection:`Inventory app --> Operations --> Internal`
-and select the checkboxes next to the pick operations that are to be added to the batch.
+   :alt: Assign a package to "Destination Package" field.
 
-After the pick operations are selected, navigate to the :guilabel:`Actions ⚙️ (gear)` icon button, and
-select :guilabel:`Add to batch` from the list of drop-down menu options.
-
-.. image:: package/create-batch.png
-   :align: center
-   :alt: Add pickings to batch.
+   Twelve units of `Acoustic Bloc Screen` are placed in `PACK0000001`.
 
-Doing so opens an :guilabel:`Add to batch` pop-up window. Click :guilabel:`Confirm` to finish
-creating the batch.
+Then, specify the quantity of items to go into the package in the :guilabel:`Done` field. Repeat the
+above steps to place the :guilabel:`Product` in different packages. Once finished, click
+:guilabel:`Confirm` to close the window.
 
 .. seealso::
-   :ref:`More info about the fields in the 'Add to batch' window
-   <inventory/warehouses_storage/add-batch-transfers>`
-
-Pack items
-----------
-
-After creating the batch, open the batch transfer (:menuselection:`Inventory app --> Operations -->
-Batch Transfers`), and select a reusable package (created in the :ref:`previous step
-<inventory/warehouses_storage/package-use>`) in the :guilabel:`Destination Package` field to place
-items in the package.
-
-.. example::
-   The reusable package, `CLUSTER-PACK-1`, is assigned to the :guilabel:`Destination Package` field
-   of all items in the batch transfer, `BATCH/00003`.
-
-  .. image:: package/assign-package.png
-     :align: center
-     :alt: Assign pickings to a package through the "Destination Package" field.
-
-Alternatively, click the :guilabel:`Put in Pack` button to create a new package, and place all the
-items in the batch in that newly-created package.
-
-.. note::
-   Ensure the new package's :guilabel:`Package Use` field is set to :guilabel:`Reusable` by clicking
-   the package's :guilabel:`➡️ (right arrow)` icon to open the package form.
+   :doc:`Ship one order in multiple packages
+   <../../shipping_receiving/advanced_operations_shipping/multipack>`
 
-   .. image:: package/pack-internal-link.png
-      :align: center
-      :alt: Show internal link
+Put in pack
+-----------
 
-.. example::
+Alternatively, click the :guilabel:`Put in Pack` button on any transfer to create a new package, and
+place all the items in the transfer in that newly-created package.
```

Comment:
does the 'Put in Pack' button appear on any transfer or just batch? Might be nice to clarify that, in case they click a transfer and don't see the 'Put in Pack' button

Reply:
Clarified! :D